app/components/createService/createservice.html

```diff
@@ -376,7 +376,7 @@
             <form class="form-horizontal" style="margin-top: 15px;">
               <!-- parallelism-input -->
```

Comment:
Could you add a "Update config" section before this input as it is done in the Resources & Placement section?